### ISSUE TITLE
iOS version?

### ISSUE THREAD

**Issue by Zorgatone** [moust/cordova-plugin-videoplayer#23]
iOS version?

**Comment by moust** [comment]
No, you can perform it with simple HTML5 video element and the Cordova's MediaPlaybackRequiresUserAction configuration option to false.

**Comment by Zorgatone** [comment]
Really? I will try

**Comment by Zorgatone** [comment]
Well, I was hoping for something that would support RTSP on iOS.. The default player doesn't play RTSP

**Comment by Zorgatone** [comment]
@moust: Does this plugin allow to play RTSP with basic authentication?

If I open a url like this in the system player it won't work: `rtsp://user:pass@my-ip/cam1`

**Comment by moust** [comment]
From what I know, Android's MediaPlayer element used in this plugin support RTSP stream but not with authentification.

**Comment by Zorgatone** [comment]
That's unfortunate. I think I will have to search for a plugin/library that does the player, authentication, decoding and all kinds of dirty stuff.

For example I saw there was VLC library that can be used with iOS (which also doesn't support RSTP out of the box)

**Comment by pnaylor** [comment]
> No, you can perform it with simple HTML5 video element and the Cordova's MediaPlaybackRequiresUserAction configuration option to false.

Will this approach still work with files stored locally as well? By setting the video source to the file's `toInternalURL`?

Thank you!
